src/App.js

```diff
@@ -62,6 +63,11 @@ export default function App() {
   const [showSelector, setShowSelector] = useState(false)
   const persona = personas.find(({ id }) => id === current)
 
+  const resetDemonstrator = (e) => {
+    e.preventDefault()
+    window.location.replace('/')
```

Comment:
why not `wndow.reload()`, https://developer.mozilla.org/en-US/docs/Web/API/Location/reload as it does not cache, e.g. skips cache as per MDN less characters, does not stop me from approving tho

Reply:
I think we want the redirect to `/` which wouldn't happen with reload